src/components/Onboarding/OnboardingPage3.tsx

```diff
@@ -89,25 +109,79 @@ const Page3: React.FC<Props> = ({ setPage, setHomeData, homeData }) => {
               }}
               containerStyle={{ height: 40 }}
               items={[
-                { label: 'Gas', value: 'Gas' },
-                { label: 'Electric', value: 'Electric' },
-                { label: 'Propane', value: 'Propane' },
-                { label: 'Other', value: 'Other' },
+                { label: 'Less than 5', value: AgeType.LESS_THAN_5 },
+                { label: 'More than 5', value: AgeType.MORE_THAN_5 },
+                { label: 'More than 10', value: AgeType.MORE_THAN_10 },
+                { label: 'More than 20', value: AgeType.MORE_THAN_20 },
               ]}
               dropDownStyle={{ width: '90%' }}
               itemStyle={{ justifyContent: 'flex-start', paddingLeft: '2%' }}
               placeholder=""
-              onChangeItem={item => setHeatingFuelType(item.value)}
-              defaultValue={homeData.fuelType}
+              onChangeItem={item => setHeaterAge(item.value)}
+              onOpen={() => setPicker1(true)}
+              onClose={() => setPicker1(false)}
             />
-            <Text style={styles.formComponent}>Age of Heater</Text>
+            <Text style={styles.formComponent}>Water Heater Fuel Type</Text>
+            {!picker1 ? (
+              <DropDownPicker
+                style={{
+                  alignSelf: 'center',
+                  borderColor: 'gray',
+                  borderWidth: 1,
+                  width: '90%',
+                  borderRadius: 5,
+                  paddingLeft: '2.5%',
+                }}
```

Comment:
nit: all of these styles should probably be moved to a StyleSheet, especially since they are reused exactly in the DropDownPicker below

Reply:
Definitely agree.

Reply:
I've refactored all of the repeated stylings.

Reply:
ok great.